tests/deployments/test_cli.py

```diff
@@ -86,10 +86,11 @@ def test_predict(tmpdir):
     temp_input_file_path = tmpdir.join("input.json").strpath
     with open(temp_input_file_path, "w+t") as temp_input_file:
         temp_input_file.write('{"data": [5000]}')
-        temp_input_file.seek(0)
+
         runner = CliRunner()
         res = runner.invoke(
-            cli.predict, ["--target", f_target, "--name", f_name, "--input-path", temp_input_file]
+            cli.predict,
+            ["--target", f_target, "--name", f_name, "--input-path", temp_input_file_path],
```

Comment:
I haven't figured out why this test started failing, but it does seem related to `click`